client/my-sites/email/email-management/email-home.jsx

```diff
@@ -82,7 +83,7 @@ class EmailManagementHome extends Component {
 
 			if ( ! domainHasEmail( selectedDomain ) ) {
 				return (
-					<EmailProvidersComparisonStacked
+					<EmailProvidersStackedComparison
 						comparisonContext="email-home-selected-domain"
 						selectedDomainName={ selectedDomainName }
 						source={ source }
```

Comment:
Should we also enclose that component with a page component like you did below?

```javascript
return (
  <>
    <PageViewTracker
      path={ emailManagement( ':site', selectedDomainName ?? null ) }
      title={ 'Email Home' }
      properties={ { source, context: 'email-home-selected-domain' } }
    />
    
    <EmailProvidersStackedComparison
      comparisonContext="email-home-selected-domain"
      selectedDomainName={ selectedDomainName }
      source={ source }
    />
  </>
);
```

Reply:
Yes please

Reply:
Done